packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx

```diff
@@ -318,6 +319,26 @@ const UserReviewFormComponent = ({
 
   const userRole = userRoles.find(({ id }) => id === formData.role)
 
+  const handleSubmit = () => {
```

Comment:
[This](https://github.com/opencrvs/opencrvs-core/blob/develop/packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx#L260) shouldn't have been removed in the first place.